CHANGELOG.md

```diff
@@ -1,5 +1,17 @@
 # Cromwell Change Log
 
+## 30 Release Notes
+
+### Other changes
+
+* **Database**  
+If desired, the metadata may be optionally stored in a separate SQL database than the of the operations data. See the
```

Comment:
I'd phrase this as "You have the option of storing the metadata in a separate SQL database than the *<database?>* containing the operations data."